docs/spec/src/protocol/contracts.md

```diff
@@ -1,27 +1,30 @@
 # EigenDA Managed Contracts
 
-This page describe EigenDA contracts that are manage by EigenDA related actors (see the exact [roles](#governance-roles)). For EigenDA-related contracts that are managed by rollups, see the [rollup managed contracts](../integration/contracts.md) page.
+This page describes EigenDA contracts that are managed by EigenDA related actors (see the exact [roles](#governance-roles)). For EigenDA-related contracts that are managed by rollups, see the [rollup managed contracts](../integration/spec/4-contracts.md) page.
+
+> Warning: This page is incomplete and a work in progress as we are undergoing refactors of our contracts as well as some protocol upgrades. The details will change, but the information contained here should at least help to understand the important concepts.
 
 ## Middlewares Contracts
 
+We make use of eigenlayer-middleware contracts, which are fully documented [here](https://github.com/Layr-Labs/eigenlayer-middleware/tree/dev/docs).
+
 ## EigenDA Specific Contracts
 
 <!-- Section copied over from https://www.notion.so/eigen-labs/EigenDA-V2-Integration-Spec-12d13c11c3e0800e8968f31ef2c6a2b3?pvs=4#18513c11c3e08058a034ddc9523a3197 -->
 <!-- TODO: arch to review and update -->
 
-The smart contracts can be found [here](https://github.com/Layr-Labs/eigenda/tree/master/contracts/src/core).
-
-
+The smart contracts can be found in our [repo](https://github.com/Layr-Labs/eigenda/tree/master/contracts/src/core), and the deployment addresses on different chains can be found in the [Networks](https://docs.eigenda.xyz/networks/mainnet#contract-addresses) section of our docs.
 
-#### EigenDACertVerifier
+The smart contracts can be found [here](https://github.com/Layr-Labs/eigenda/tree/master/contracts/src/core).
```

Comment:
This new line is redundant: smart contracts location is already given above

Reply:
[55540d6](https://github.com/Layr-Labs/eigenda/pull/1637/commits/55540d6a43c5c8e68d445d7ee7a05381c82a8334)